src/components/FlatList/MVCPFlatList.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable es/no-optional-chaining, es/no-nullish-coalescing-operators, react/prop-types */
```

Comment:
Can we try to resolve some of these lints instead of just disabling them? Or you could write this in TS if you'd like

Reply:
I wanted to keep the code as close as possible as the PR it is based on so I disabled the lint rules instead of trying to fix it. I think this is ok since this code is temporary until proper maintainVisibleContentPosition is merged in rn-web.

Reply:
Cool thanks for the context - agree with that